scripts/build_nginx_image.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
I might move this into the same folder as the nginx Dockerfile as this is specific to images created with it.